src/nss-systemd/nss-systemd.c

```diff
@@ -312,16 +312,16 @@ enum nss_status _nss_systemd_endgrent(void) {
 }
 
 enum nss_status _nss_systemd_setpwent(int stayopen) {
+        int r;
+
         PROTECT_ERRNO;
         NSS_ENTRYPOINT_BEGIN;
 
         if (_nss_systemd_is_blocked())
                 return NSS_STATUS_NOTFOUND;
 
-        _cleanup_(pthread_mutex_unlock_assertp) pthread_mutex_t *_l = NULL;
-        int r;
-
-        _l = pthread_mutex_lock_assert(&getpwent_data.mutex);
+        _cleanup_(pthread_mutex_unlock_assertp) pthread_mutex_t *_l =
```

Comment:
20:49:35     archlinux_systemd_ci: ../build/src/nss-systemd/nss-systemd.c:323:66: error: unused variable '_l' [-Werror,-Wunused-variable]
20:49:35     archlinux_systemd_ci:         _cleanup_(pthread_mutex_unlock_assertp) pthread_mutex_t *_l =

Reply:
Yeah, using a separate variable here is a trick to trick clang into thinking that the variable is used.